ci/pipelines/deploy-to-test.yml

```diff
@@ -1571,6 +1572,31 @@ jobs:
         params:
           APP_NAME: toolbox
           <<: *wait_for_deploy_params
+      - task: run-toolbox-metric-tests
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: governmentdigitalservice/pay-node-runner
+              tag: node16
+          inputs:
+            - name: pay-ci
+          params:
+            TEST_METRIC_IMAGE_TAG: candidate
+            TEST_METRIC_ECS_SERVICE: toolbox
+            # TODO: find out how to get endpoint and creds
```

Comment:
There's a rather helpful yaml anchor which will fill all this in for you `&aws_test_config` which has all these in:
```
   1 aws_test_config: &aws_test_config
   2   aws_access_key_id: ((readonly_access_key_id))
   3   aws_secret_access_key: ((readonly_secret_access_key))
   4   aws_session_token: ((readonly_session_token))
   5   # Need to create the role for the concourse user to assume in the test account
   6   aws_role_arn: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
   7   # Hardcode the test account registry ID for now. Needs to be a string, not a number
   8   aws_ecr_registry_id: "((pay_aws_test_account_id))"
   9   aws_region: eu-west-1
   ```
   in

Reply:
Endpoint is an interesting one. I think it's probably ok to hard code it which we have done in some other places. The workgroup url can't change or we're right up the swanny :D

Reply:
That's what I thought. I found the endpoint in the Grafana datasource after I put in the "find out how to get endpoint" comment. I thought we'd need to assume a role rather than getting access keys.

Reply:
The role the concourse worker has doesn't have permission to query prometheus, but we also don't have a role for it to assume to query the metrics 🤔 
I think doing either would be ok, and it's much easier to give concourse the ability to query the prometheus metrics than making a new role just for it.

The concourse worker role is created by this module usage: https://github.com/alphagov/pay-concourse-deployment/blob/main/terraform/deployments/pay-cd/team-pay-dev.tf#L23, you can get the role name for doing a role policy attachment by doing `module.concourse_keys.concourse_worker_iam_role_names["pay-dev"]`. There's an AWS pre-baked policy to query prometheus we can use which we [already attach to grafana here](https://github.com/alphagov/pay-concourse/blob/main/terraform/modules/concourse-monitoring/iam.tf#L326-L333)

Reply:
API calls must be signed, as-per https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-onboard-query-APIs.html. There's no aws-sdk helper for it, you're (to some degree) on your own.

I've added signing to the script, and its `aws-crt` dependency to the node-runner container in #936. Looking at similar jobs, I think the creds required for signing will be passed through to the container.